src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputRightCurlyDoWhileAlone.java

```diff
@@ -59,12 +59,12 @@ public void foo4() {
 
     public void foo5() {
         do {
-        } // ok - for alone
+        } // warn
```

Comment:
we need to change name of file Input files, it become very confusing.
name should be just be "InputRighCurlyDoWhile" no "Alone" or "Same" suffixes.

Reply:
Done.